src/main/scala/vectorpipe/vectortile/Pipeline.scala

```diff
@@ -0,0 +1,101 @@
+package vectorpipe.vectortile
+
+import geotrellis.vector.Geometry
+import geotrellis.spark.SpatialKey
+import geotrellis.spark.tiling._
+import org.apache.spark.sql.{DataFrame, Row}
+import org.locationtech.jts.{geom => jts}
+
+/**
+ * The interface governing the transformation from processed OSM dataframes to
+ * vector tiles.
+ *
+ * After loading an OSM source and transforming via [[OSM.toGeometry]] and any
+ * filtering/modification steps, one may want to output a set of vector tiles
+ * from the remaining geometry and metadata.  The vector tile output process is
```

Comment:
So you're saying that this could also be used for other geometries that can be loaded into a DataFrame?

Reply:
It seems I _didn't_ say that, but I do think there's no reason that this wouldn't also work for non-OSM-derived JTS frames.  I wonder if there's a common infrastructure in here that would work for general `Dataset`s instead, or if it's sufficient to handle `DataFame`s only.